mobile/lib/main.dart

```diff
@@ -6,12 +6,15 @@ import 'package:app/widgets/widgets.dart';
 import 'package:firebase_core/firebase_core.dart';
 import 'package:firebase_dynamic_links/firebase_dynamic_links.dart';
 import 'package:flutter/material.dart';
-
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
 import 'firebase_options.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
+  await Firebase.initializeApp();
 
+// Pass all uncaught "fatal" errors from the framework to Crashlytics
+  FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterFatalError;
```

Comment:
You can place this in the [initializeMainMethod()](https://github.com/airqo-platform/AirQo-frontend/blob/0c04447e5ef6878d9d4a3dae7a0387e79375047a/mobile/lib/main_common.dart#L124) method